apps/omg_watcher/lib/web/error_handler.ex

```diff
@@ -0,0 +1,80 @@
+# Copyright 2018 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.Watcher.Web.ErrorHandler do
+  @moduledoc """
+  Handles API errors by mapping the error to its response code and description.
+  """
+  alias OMG.Watcher.Web.Serializer
+
+  import Plug.Conn, only: [halt: 1]
+  import Phoenix.Controller, only: [json: 2]
+
+  @errors %{
+    invalid_challenge_of_exit: %{
+      code: "challenge:invalid",
+      description: "The challenge of particular exit is invalid"
```

Comment:
I'm thinking about reducing the number of future support calls ;) Can we make descriptions more descriptive? E.g. why challenge is invalid? Maybe it's simple answer...